### PR TITLE
Update library versions

### PR DESCRIPTION
**Pull request by conniey**
* Updates Event Hubs and Checkpoint Store versions. So we can release a minor version fix.

### CODE REVIEW

File: sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 5.1.0-beta.1 (Unreleased)
-
+## 5.0.2 (2020-02-13)
+- Fix bug where consumers would not be able to block in their consuming code.
```
Comment (srnagar) [inline sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md]: Is this only for the consumer or producer too?

File: sdk/search/azure-search/pom.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```
Comment (srnagar) [inline sdk/search/azure-search/pom.xml]: This is azure-search. Don't think this should be updated.

File: sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/pom.xml [hunk 3]
```diff
@@ -40,7 +40,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-messaging-eventhubs</artifactId>
-      <version>5.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;current} -->
+      <version>5.0.2</version> <!-- {x-version-update;com.azure:azure-messaging-eventhubs;current} -->
```
Comment (srnagar) [inline sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/pom.xml]: Interesting, we have already update storage blob version too.

File: sdk/search/azure-search/pom.xml (hunk 2 above)
Comment (conniey) [reply sdk/search/azure-search/pom.xml]: The build was failing before this change because it was complaining that a version wasn't updated.

Comment (JimSuplizio) [reply sdk/search/azure-search/pom.xml]: This was fixed yesterday with https://github.com/Azure/azure-sdk-for-java/pull/8188. This mismatch was the result of two independency PRs being auto-merged. You'll have to rebase and push another commit without this file.

**Pull request merged**